### DISCUSSION TITLE
What's the alternative to `get_engine(bind="replica")` in versions 3+?

### DISCUSSION THREAD

**Discussion by andrii-korotkov-verkada** [pallets-eco/flask-sqlalchemy#1374] [Q&A] [answered]
Hey. The database is set up to have write and read replicas. There are some methods used on read replica engine, such as `get_engine(bind="replica").pool.overflow` on `SQLAlchemy` object. But `get_engine` is deprecated in v3. What's the alternative way to achieve the same goal?

**Comment by davidism** [comment, marked as answer]
The deprecation warning shows the following message:

> 'get_engine' is deprecated and will be removed in Flask-SQLAlchemy 3.2. Use 'engine' or 'engines[key]' instead. If you're using Flask-Migrate or Alembic, you'll need to update your 'env.py' file.

Therefore: `db.engines["replica"]`
